ui/.eslintrc.js

```diff
@@ -39,10 +39,6 @@ module.exports = {
     "keyword-spacing" : "error",
     "no-trailing-spaces" : "error",
     "object-curly-spacing" : ["error", "always"],
-    "arrow-spacing" : ["error", {
-      "before" : true,
-      "after" : true,
-    }],
```

Comment:
This is a "good-to-have" check I think. Keeps code clean, and well oriented.

Reply:
dont worry @Nikhil-Ladha I hadn't removed it , It was declared same two times , so I removed at one place :smiley: